pennylane/ops/cv.py

```diff
@@ -132,7 +132,8 @@ class Rotation(CVOperation):
 
     * Number of wires: 1
     * Number of parameters: 1
-    * Gradient recipe: :math:`\frac{d}{dr}R(r) = \frac{1}{2} \left[R(\phi+\pi/2) - R(\phi-\pi/2)\right]`
+    * Gradient recipe: :math:`\frac{d}{dr}f(R(r)) = \frac{1}{2} \left[f(R(\phi+\pi/2)) - f(R(\phi-\pi/2))\right]`
+      where :math:`f` is an expectation value depending on :math:`R(r)`.
```

Comment:
Should we explicitly state :math:\`f(R(r))=\langle \psi(r) | \hat{B} | \psi(r)\rangle\` for some expectation value \`\hat{B}\`, where \`\psi(r)\` is the final state of the circuit?

Reply:
I thought about this, but then it would become very crowded in there, just because you would have to repeat it at every entry that has a gradient recipe. I think this is a good compromise.

Reply:
Perhaps if the notation/terminology is detailed once, somewhere in the module docstring, so that it appears on the top of the page before the reader gets to the individual operation docstrings?

Reply:
That would be an improvement, but I think the problem is that if people hit up 'RX' for example they get there and probably won't read the module description.

Another possibility would be to add a link 'learn more about gradient recipes' that links to the explanation in the overview. In this case one could also think about just giving the gradient recipe as `{1/2s, s}`, for example.